recipes/install.rb

```diff
@@ -48,3 +47,11 @@
     cwd     "#{node[:valhalla][:src_dir]}/#{repo}"
   end
 end
+
+# link the lua transforms from the checkout
+%w(vertices edges admins).each do |lua|
```

Comment:
I thought it was better to have here, but if you feel it should be moved we can.  I was thinking scripts take over after all is set up and everything is in a clean state.